src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,19 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  query: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+export const MoviesList: React.FC<Props> = ({ movies, query }) => {
+  const visibleMovies = movies
+    .filter(movie => movie.title.toLowerCase().includes(query.toLowerCase())
+        || movie.description.includes(query.toLowerCase()));
```

Comment:
query.toLowerCase() better to pass in a separate variable not to calculate it in every iteration